apps/src/lib/levelbuilder/script-editor/LessonGroupCard.jsx

```diff
@@ -69,6 +69,7 @@ class LessonGroupCard extends Component {
     lessonGroupMetrics: PropTypes.object,
     setTargetLessonGroup: PropTypes.func,
     targetLessonGroupPos: PropTypes.number,
+    lessonKeys: PropTypes.array,
```

Comment:
looks like this should be required, since we are accessing it unconditionally below